### DISCUSSION TITLE
Daikin ECH2O - external unit ERWQ02AAV3

### DISCUSSION THREAD

**Discussion by nosfaratu** [raomin/ESPAltherma#255] [Q&A]
good night guys!

I have a daikin ECH2O external unit ERWQ02AAV3, and I would like to know if someone has already done the integration using this method. It's just that some flyers say it's Altherma.

I can't find any board information in the instruction manuals.
And not to be opening the machine in vain, first I wanted to know if someone has already tested this outdoor unit. Or if it is possible to use.

I have all the material, I just need to know if it works or not

Thanks!

**Comment by KalaNagHTD** [comment]
It will work! The connection has to be on the board A1P of the internal unit (HPSU/ECH2O). The A1P is hidden behind the BM1 board.

https://github.com/raomin/ESPAltherma#8-pin-x10a-connection

**Comment by nosfaratu** [comment]
![PXL_20230224_082048693](https://user-images.githubusercontent.com/3196254/221129114-0afc77d1-1525-481a-a8d7-cc110965ed44.jpg)
Thanks!  The outside unit is ok.  This is my indoor unit.  Does it work too?  Is everything ok with you ok for me to do the tomorrow?  :D
And the outside unit 
![IMG20230223183424](https://user-images.githubusercontent.com/3196254/221132514-3b58f91d-268a-4c39-93a0-ab5a1a815624.jpg)

**Reply by KalaNagHTD** [reply]
I am very sure it will work! You have ro find the A1P board with the X10A connector inside the electronic box of the inside unit.

**Reply by nosfaratu** [reply]
ahah more simple its impossible!
![IMG20230224222917](https://user-images.githubusercontent.com/3196254/221307196-6ecf5140-69ee-4f22-b636-f26649f1b8e0.jpg)

I will configure the ESP32 and test!

**Comment by nosfaratu** [comment]
I can't configure this..
As much as I try to insert the includes, I don't see any sensors..
I am using a wroom esp32 devkit.
in mqtt itself, the message appears that there is no include.
can you provide your config to test?

**Comment by KalaNagHTD** [comment]
Your problem has nothing to do with your Daikin hardware! You first have to get your ESPAltherma running correctly.

**Comment by nosfaratu** [comment]
Yes, I know it's the configuration.  I'm having problems with the configuration of the program..:/

**Comment by nosfaratu** [comment]
Esp well configured, give me the error "Time out! Check connection"
![IMG20230224222917](https://user-images.githubusercontent.com/3196254/221354473-b5976fab-65f3-46b9-8c91-7d1540733765.jpg)
Is the beginning of the numbering correct?
I've changed cables twice and nothing.

**Reply by KalaNagHTD** [reply]
Correct!

![3193D366-5E83-4952-BAD3-9DFA037D9CBB](https://user-images.githubusercontent.com/77481230/221358253-2f619dc2-f1b5-418f-8f59-29edbf3df193.png)

**Comment by nosfaratu** [comment]
Well.. I managed to get this working!
My altherma despite saying daikin, is Rotex, so I had to #define PROTOCOL 'S' Also select the def/PROTOCOL_S_ROTEX.h .
and had the wrong rx and tx connections..

The sensors that appear to me are just these.
![altherma](https://user-images.githubusercontent.com/3196254/221365015-b001b722-62d3-4ded-b6c5-c6974c099829.png)

Is there no possibility to add more?
